.github/fixtures/new-fixture-template/cliff.toml

```diff
@@ -19,15 +18,16 @@ body = """
     {% endfor %}
 {% endfor %}\n
 """
-# template for the changelog footer
-footer = """
+# A Tera template to be rendered as the changelog's footer (see https://keats.github.io/tera/docs/#introduction).
+footer_template = """
 <!-- generated by git-cliff -->
 """
-# remove the leading and trailing whitespace from the templates
-trim = true
+# Whether to remove leading and trailing whitespaces from all lines of the changelog's body.
+trim_body_whitespace = true
```

Comment:
doesn't this also trim new lines?
Maybe `trim_body` is enough as a name. 🤔

Reply:
Unless I misunderstand the code in [git-cliff-core/src/template.rs:34](https://github.com/Cyclonit/git-cliff/blob/f69dfe42d1c408140b871df126bcfbd5a668f469/git-cliff-core/src/template.rs#L37), this option does not remove newlines.